src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double firstLeg;
+    private final double secondLeg;
+
+    RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return firstLeg * secondLeg * 0.5;
```

Comment:
All magic numbers in your code should be constants. The value '0.5' used in the getArea() method should be extracted to a constant with a descriptive name.

Reply:
0.5 here is a part of formule to find an area